source/docs/05-mensural.xml

```diff
@@ -92,8 +92,8 @@
             <p>This chapter describes the module for encoding mensural notation from the late 13th century to about 1600. Historically, mensural notation preceded the development of Common Music Notation (CMN) and it included a wide range of features that persist in CMN and that can be encoded in a standard manner in MEI. In mensural notation, pitches are notated as in CMN, leaving out here the major exception of <hi rend="italic">musica ficta</hi>. The pitch is given by the position of the note on the staff and the current clef as in CMN, and the mensural module introduces no modification to MEI regarding how pitches are encoded.</p>
-            <p>There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main particularities is that the actual duration of a note is not given only by its symbol but also by position and the context in which the symbol appears. The general context is given by one of the 16 mensural <hi rend="italic">species</hi> provide four levels of division: <hi rend="italic">modus major</hi>, <hi rend="italic">modus minor</hi>, <hi rend="italic">tempus</hi> and <hi rend="italic">prolatio</hi>. Depending on the context, certain rules must be applied in order to determine the duration of a note. In these cases, encoding both the sign and its actual duration is highly desirable.</p>
-            <p>Another particularity of mensural notation is the use of proportions that are indicated by numeric proportions or by specific mensuration signs. The proportions indicate that the durations have to be modified accordingly and they can be combined in a very complex manner. Over time, proportions and mensuration signs were simplified and became time signatures in CMN. The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module.</p>
+            <p>There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main particularities is that the actual duration of a note is not given only by its symbol but also by the position and the context in which the symbol appears. The general context is given by one of the 16 mensural <hi rend="italic">species</hi> provided by the four levels of division: <hi rend="italic">modus major</hi>, <hi rend="italic">modus minor</hi>, <hi rend="italic">tempus</hi> and <hi rend="italic">prolatio</hi>. Then, depending on the specific context where the note is positioned, certain rules must be applied in order to determine the duration of a note. In these cases, encoding both the sign and its actual duration is highly desirable.</p>
```

Comment:
```suggestion
            <p>There are a number of differences, however, in the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main differences is that the duration of a note is not determined by its symbol, but also by the position and the context in which the symbol appears in relation to other notes and rests in the same voice. The context is given by one of the 16 mensural <hi rend="italic">species</hi> provided by four levels of division: <hi rend="italic">modus major</hi>, <hi rend="italic">modus minor</hi>, <hi rend="italic">tempus</hi> and <hi rend="italic">prolatio</hi>. Depending on the specific context where the note is positioned, certain rules must be applied in order to determine the duration of a note. In these cases, encoding both the sign and its actual duration is highly recommended.</p>
```

Reply:
"...encoding both the sign and its actual duration..." I'm not sure what this means? And no examples are provided.

Reply:
I understand what you mean by these corrections. Still, I think the original text was referring to the "general context" (the mensuration) as the "context", and it was referring to the "specific context" (notes preceding and following) as the "position" of the note. So, I will accept these corrections and edit the text a bit more so that it becomes clear.

And regarding your comment:
>"...encoding both the sign and its actual duration..." I'm not sure what this means? And no examples are provided.

The example is provided later on in the section about durations. I will try to make that clear.